### ISSUE TITLE
A11y: Heading order on the 526ez review-and-submit page is out of order for rated disabilities card

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/va.gov-team#106385]
## Issue Description
When a veteran has a rated disability, it shows up on the review and submit page. The "Existing Conditions" heading is H4, while the disability card heading is H3. The H3 is "under" the H4, and should be an H5 instead. When looking at the code, they aren't actually nested under each other, so it's not triggering the accessibility errors, but visually it's "under."
<img width="1009" alt="Image" src="https://github.com/user-attachments/assets/57fce52d-6581-4f3a-acf5-7c86e8e43db0" />

---
## Tasks
- [ ] Change the disability card level to be H5.

## Acceptance Criteria
- [ ] Disability card will use an h5.

---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)
